### ISSUE TITLE
Fish shows (Variable:) instead of (Variable) when trying to complete a variable

### ISSUE THREAD

**Issue by ariasuni** [fish-shell/fish-shell#4767]
I use fish 2.7.1.

When I try to complete `$`, I get a list of variables like this:

```
$_                           (Variable:)  $fish_color_cwd              (Variable:) […]
```

This should displays like this:

```
$_                           (Variable)   $fish_color_cwd              (Variable) […]
```

**Comment by faho** [comment]
Are all of these empty?

What it should display is "Variable: VALUE" (i.e. with the actual value). And when VALUE is empty, that becomes "Variable:".

**Comment by ariasuni** [comment]
No, they are set (I use default configuration so this is not me messing with config files).

For example, `$FISH_VERSION` is set to `2.7.1` but completion shows `(Variable:)`.

I noticed that launching fish with `LANG=en_US.UTF-8 fish` fixes the problem.

**Comment by faho** [comment]
Ah, okay.

This is an issue with the french translations in 2.7.1. This one looks like:

```
msgid "Variable: %ls"
msgstr "Variable: "
```

Obviously it's missing the "%ls" specifier.

It's been added in one of the PRs since then, and will be released in 3.0 next month.

Sorry for the inconvenience!
